src/pyproject_fmt/cli.py

```diff
@@ -60,6 +56,8 @@ def pyproject_toml_path_creator(argument: str) -> Path:
     :param argument: the string argument passed in
     :return: the pyproject.toml path
     """
+    if argument == "-":
```

Comment:
Path("-") makes no sense, perhaps this should return None instead for this cases.

Reply:
Replaced `Path("-")` with `None`.